.github/workflows/publish-python.yml

```diff
@@ -0,0 +1,158 @@
+name: Publish Python Bindings
+on:
+  workflow_call:
+    inputs:
+      ref:
+        description: 'commit/tag/branch reference'
+        required: true
+        type: string
+      package-version:
+        description: 'version for the python package (MAJOR.MINOR.BUILD)'
+        required: true
+        type: string
+      publish:
+        description: 'value indicating whether to publish to pypi.'
+        required: true
+        type: boolean
+        default: false
+    secrets:
+      PYPI_API_TOKEN:
+        description: 'api token to authenticate to pypi'
+        required: true
+
+jobs:
+  build-macos-wheels:
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        python: ["3.8", "3.9", "3.10"]
+    steps:
+      - name: Checkout breez-sdk repo
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.ref || github.sha }}
+
+      - name: "Install Python"
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python }}
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: sdk-bindings-darwin-universal
+          path: libs/sdk-bindings/bindings-python/src/breez_sdk
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: bindings-python
+          path: libs/sdk-bindings/bindings-python/src/breez_sdk
+
+      - name: Clean up downloaded files
+        run: |
+          rm -f libs/sdk-bindings/bindings-python/src/breez_sdk/*.a
+          ls -R libs/sdk-bindings/bindings-python
+
+      - name: Update package version
+        if: ${{ inputs.package-version }}
+        working-directory: libs/sdk-bindings/bindings-python
+        run: sed -i.bak -e 's/    version=".*",/    version="${{ inputs.package-version }}",/' setup.py
```

Comment:
One last question: the `-i.bak` creates a `setup.py.bak` file. Does the `setup.py.bak` file get published in the wheel? Do we have to remove it?